<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="ApiTest"
           tests="20"
           failures="7"
           errors="0"
           skipped="0"
           time="2.376">
  <properties />
            <testcase name="apiErrorBlankCity" time="0.031" classname="automationFramework.TestCases.ApiCustomerPost">
                <failure type="java.lang.AssertionError"
                  message="expected [If address is provided, city is required.] but found [Field is blank.]"
              >
<![CDATA[java.lang.AssertionError: expected [If address is provided, city is required.] but found [Field is blank.]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertEquals(Assert.java:123)
  org.testng.Assert.assertEquals(Assert.java:176)
  org.testng.Assert.assertEquals(Assert.java:186)
  automationFramework.TestCases.ApiCustomerPost.checkError(ApiCustomerPost.java:374)
  automationFramework.TestCases.ApiCustomerPost.apiErrorBlankCity(ApiCustomerPost.java:233)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
  org.testng.TestNG.run(TestNG.java:1031)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:177)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:105)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
  org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
  org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:103)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:74)
]]>
      </failure>

          </testcase>
            <testcase name="apiErrorInvalidCustomerType" time="0.029" classname="automationFramework.TestCases.ApiCustomerPost">
                <failure type="java.lang.AssertionError"
                  message="expected [Invalid customer type.] but found [Invalid value.]"
              >
<![CDATA[java.lang.AssertionError: expected [Invalid customer type.] but found [Invalid value.]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertEquals(Assert.java:123)
  org.testng.Assert.assertEquals(Assert.java:176)
  org.testng.Assert.assertEquals(Assert.java:186)
  automationFramework.TestCases.ApiCustomerPost.checkError(ApiCustomerPost.java:374)
  automationFramework.TestCases.ApiCustomerPost.apiErrorInvalidCustomerType(ApiCustomerPost.java:65)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
  org.testng.TestNG.run(TestNG.java:1031)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:177)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:105)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
  org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
  org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:103)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:74)
]]>
      </failure>

          </testcase>
            <testcase name="apiErrorBlankAddress" time="0.043" classname="automationFramework.TestCases.ApiCustomerPost">
                <failure type="java.lang.AssertionError"
                  message="expected [If address is provided, address1 is required.] but found [Field is blank.]"
              >
<![CDATA[java.lang.AssertionError: expected [If address is provided, address1 is required.] but found [Field is blank.]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertEquals(Assert.java:123)
  org.testng.Assert.assertEquals(Assert.java:176)
  org.testng.Assert.assertEquals(Assert.java:186)
  automationFramework.TestCases.ApiCustomerPost.checkError(ApiCustomerPost.java:374)
  automationFramework.TestCases.ApiCustomerPost.apiErrorBlankAddress(ApiCustomerPost.java:198)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
  org.testng.TestNG.run(TestNG.java:1031)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:177)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:105)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
  org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
  org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:103)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:74)
]]>
      </failure>

          </testcase>
            <testcase name="apiErrorBlankCountry" time="0.029" classname="automationFramework.TestCases.ApiCustomerPost">
                <failure type="java.lang.AssertionError"
                  message="expected [If address is provided, country is required.] but found [Field is blank.]"
              >
<![CDATA[java.lang.AssertionError: expected [If address is provided, country is required.] but found [Field is blank.]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertEquals(Assert.java:123)
  org.testng.Assert.assertEquals(Assert.java:176)
  org.testng.Assert.assertEquals(Assert.java:186)
  automationFramework.TestCases.ApiCustomerPost.checkError(ApiCustomerPost.java:374)
  automationFramework.TestCases.ApiCustomerPost.apiErrorBlankCountry(ApiCustomerPost.java:255)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
  org.testng.TestNG.run(TestNG.java:1031)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:177)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:105)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
  org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
  org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:103)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:74)
]]>
      </failure>

          </testcase>
            <testcase name="apiErrorAddressId" time="1.592" classname="automationFramework.TestCases.ApiCustomerPost">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  automationFramework.TestCases.ApiCustomerPost.checkError(ApiCustomerPost.java:372)
  automationFramework.TestCases.ApiCustomerPost.apiErrorAddressId(ApiCustomerPost.java:187)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
  org.testng.TestNG.run(TestNG.java:1031)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:177)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:105)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
  org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
  org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:103)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:74)
]]>
      </failure>

          </testcase>
            <testcase name="apiErrorInvalidContactType" time="0.045" classname="automationFramework.TestCases.ApiCustomerPost">
                <failure type="java.lang.AssertionError"
                  message="expected [Invalid contact type.] but found [Invalid value.]"
              >
<![CDATA[java.lang.AssertionError: expected [Invalid contact type.] but found [Invalid value.]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertEquals(Assert.java:123)
  org.testng.Assert.assertEquals(Assert.java:176)
  org.testng.Assert.assertEquals(Assert.java:186)
  automationFramework.TestCases.ApiCustomerPost.checkError(ApiCustomerPost.java:374)
  automationFramework.TestCases.ApiCustomerPost.apiErrorInvalidContactType(ApiCustomerPost.java:86)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
  org.testng.TestNG.run(TestNG.java:1031)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:177)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:105)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
  org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
  org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:103)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:74)
]]>
      </failure>

          </testcase>
            <testcase name="apiErrorInvalidSuffixId" time="0.054" classname="automationFramework.TestCases.ApiCustomerPost">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  automationFramework.TestCases.ApiCustomerPost.checkError(ApiCustomerPost.java:372)
  automationFramework.TestCases.ApiCustomerPost.apiErrorInvalidSuffixId(ApiCustomerPost.java:176)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
  org.testng.TestNG.run(TestNG.java:1031)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:177)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:105)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
  org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
  org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:103)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:74)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="apiErrorBlankContactName" time="0.036" classname="automationFramework.TestCases.ApiCustomerPost" />
                <testcase name="apiErrorBlankContactType" time="0.039" classname="automationFramework.TestCases.ApiCustomerPost" />
                <testcase name="apiErrorBlankCustomerType" time="0.075" classname="automationFramework.TestCases.ApiCustomerPost" />
                <testcase name="apiPostSuccess" time="0.055" classname="automationFramework.TestCases.ApiCustomerPost" />
                <testcase name="apiErrorMaxLengthContactLastName" time="0.041" classname="automationFramework.TestCases.ApiCustomerPost" />
                <testcase name="apiErrorBlankContactFirstName" time="0.052" classname="automationFramework.TestCases.ApiCustomerPost" />
                <testcase name="apiErrorMaxLengthCity" time="0.047" classname="automationFramework.TestCases.ApiCustomerPost" />
                <testcase name="apiErrorMaxLengthContactMiddleName" time="0.030" classname="automationFramework.TestCases.ApiCustomerPost" />
                <testcase name="apiErrorBlankContact" time="0.032" classname="automationFramework.TestCases.ApiCustomerPost" />
                <testcase name="apiErrorMaxLengthAddress2" time="0.030" classname="automationFramework.TestCases.ApiCustomerPost" />
                <testcase name="apiErrorMaxLengthContactFirstName" time="0.054" classname="automationFramework.TestCases.ApiCustomerPost" />
                <testcase name="apiErrorBlankContactLastName" time="0.032" classname="automationFramework.TestCases.ApiCustomerPost" />
                <testcase name="apiErrorMaxLengthAddress" time="0.030" classname="automationFramework.TestCases.ApiCustomerPost" />
      
</testsuite>
